res/openvpn-gui-res-it.rc

```diff
@@ -234,25 +234,25 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
     ICON ID_ICO_APP, ID_ICON_ABOUT, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v%ls - Interfaccia per Windows di OpenVPN\n\
+    LTEXT "OpenVPN GUI v%ls - Interfaccia di OpenVPN per Windows\n\
 Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n\
 Copyright (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n\
-Copyright (C) 2012-2022 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
+Copyright (C) 2012-2021 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
     LTEXT "https://github.com/OpenVPN/openvpn-gui/", ID_LTEXT_ABOUT2, 36, 55, 206, 50
-
-    LTEXT "OpenVPN - Un'applicazione che permette di creare tunnel sicuri su reti IP \
-con una singola porta TCP/UDP, con il supporto basato sull'autenticazione e lo scambio \
-di chiavi SSL/TLS, cifratura, autenticazione e compressione dei pacchetti.\n\
+    LTEXT "OpenVPN - Un'applicazione che permette di creare tunnel sicuri \
+in reti IP  con una singola porta TCP/UDP, con il supporto basato \
+sull'autenticazione e lo scambio di chiavi SSL/TLS, cifratura, \
+autenticazione e compressione dei pacchetti.\n\
 \n", ID_LTEXT_ABOUT3, 8, 70, 240, 64
-    LTEXT "Copyright (C) 2002-2022 OpenVPN Technologies, Inc <info@openvpn.net>\n\
```

Comment:
Here too.
You can add a fixup here -- please do not close and reopen the PR as that makes review hard. If its a minor thing like this one I can fix it too.

Thanks for the update, by the way :)